src/app/modules/shared/components/month-picker/month-picker.component.ts

```diff
@@ -40,6 +41,8 @@ export class MonthPickerComponent implements OnInit {
 
     const monthIndex = changeAction === 'add' ? 0 : 11;
     this.selectMonth(monthIndex);
+
+    this.showArrowNext = this.selectedYear < this.currentYear ?? !this.showArrowNext;
```

Comment:
I won't put `!this.showArrowNext` I'll just put `false` is better for reading.

Reply:
ok, the default value of `this.showArrowNext` is false, in this case I have to change to` true`